### PR TITLE
Landing page ver2

### PR DESCRIPTION
**Pull request by MikaelFeher**
PT Story: https://www.pivotaltracker.com/story/show/128493117

Changes proposed in this pull request:
- Adding google maps
- Adding list of cuisines
- Adding dummy input field for location

Ready for review!

### CODE REVIEW

**Comment by luciademoja** [comment]
I think this is good to merge, @diraulo @tochman anything to comment?

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -14,6 +14,6 @@
 //= require jquery_ujs
 //= require foundation
 //= require turbolinks
-//= require_tree .
+///= require_tree .
```
Comment (diraulo) [inline app/assets/javascripts/application.js:17]: @MikaelFeher why is there a third slash here?

**Comment by MikaelFeher** [comment]
@tochman added that during the process of implementing the google maps.

**Comment by tochman** [comment]
We want to control what js files get loaded by the application.js

**Comment by tochman** [comment]
Ready to go imo.

**Comment by diraulo** [comment]
@tochman if the intention was to no load all js files from the directory tree then just removing the `=` sign would do

**Pull request merged**